pro/firebase.json

```diff
@@ -1,16 +1,23 @@
 {
   "hosting": {
     "public": "build",
-    "ignore": [
-      "firebase.json",
-      "**/.*",
-      "**/node_modules/**"
-    ],
+    "ignore": ["firebase.json", "**/.*", "**/node_modules/**"],
     "rewrites": [
       {
         "source": "**",
         "destination": "/index.html"
       }
+    ],
+    "headers": [
+      {
+        "source": "**",
+        "headers": [
+          {
+            "key": "Content-Security-Policy-Report-Only",
+            "value": "style-src 'self' https://app.getbeamer.com/styles/beamer-embed.css 'unsafe-inline'; img-src 'self' test data: https://storage.googleapis.com; object-src 'none'; frame-src https://www.google.com https://data-analytics.passculture.team; script-src-elem 'self' https://www.gstatic.com https://static.hotjar.com https://app.getbeamer.com https://script.hotjar.com https://www.googletagmanager.com https://firebaseinstallations.googleapis.com https://firebaseremoteconfig.googleapis.com https://vc.hotjar.io https://www.google.com; connect-src https://firebase.googleapis.com https://firebaseinstallations.googleapis.com https://sentry.passculture.team https://backend.testing.passculture.team https://region1.google-analytics.com https://*.algolia.net https://insights.algolia.io https://vc.hotjar.io https://metrics.hotjar.io; report-uri https://sentry.passculture.team/api/2/security/?sentry_key=50f5694849704813b4154c5868b73365;"
```

Comment:
> **Compatibility Recommendation**
> Though the report-to directive is intended to replace the deprecated report-uri directive, report-to isn't supported in most browsers yet. So for compatibility with current browsers while also adding forward compatibility when browsers get report-to support, you can specify both report-uri and report-to in your Content-Security-Policy (CSP).

cf https://docs.sentry.io/security-legal-pii/security/security-policy-reporting/ et 
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/report-uri

Reply:
Oui, on a testé différents navigateurs récents, et même déprécié, `report-uri` est toujours supporté, et le sera a priori toujours pendant les prochaines semaines de surveillance d'ici qu'on modifie les vraies CSP. On pourrait mettre les deux mais on voyait pas vraiment l'intérêt

Reply:
Rectification, la directive `report-to` n'a pas été utilisée parce qu'on arrive pas à la faire fonctionner en suivant les différentes docs de l'api reporting! On a documenté ce qu'on a testé plus en détail dans le notion lié dans la PR